### PR TITLE
Make FURN compliant by migrating to 1ESPT pipelines

### PR DESCRIPTION
**Pull request by dannyvv**
### Description of changes

Update publish pipelines to be in line iwht latest complaince requirements

### Verification

ran pipelines on private branch

### CODE REVIEW

File: .ado/templates/e2e-dependency-nuget-publish.yml [hunk 1]
```diff
@@ -4,42 +4,35 @@ parameters:
...
+        workingDirectory: tester_deps/
+        displayName: 'yarn install'
+
+      # Pack the NuGet package
+      - task: CmdLine@1
+        displayName: 'Create NuGet package for tester_deps.'
+        inputs:
+          filename: nuget
+          arguments: 'pack Microsoft.FluentUI.WebDriverIO.Appium.Deps.nuspec -OutputDirectory $(Build.ArtifactStagingDirectory) -OutputFileNamesWithoutVersion -Verbosity detailed -Version $(Build.BuildNumber) -properties CommitId=$(Build.SourceVersion)'
+          workingFolder: 'tester_deps/nuget'
```
Comment (Saadnajmi) [inline .ado/templates/e2e-dependency-nuget-publish.yml:38]: Did we lose the `Nuget push` step?

Comment (Saadnajmi) [reply .ado/templates/e2e-dependency-nuget-publish.yml:38]: Same for other YML files

Comment (dannyvv) [reply .ado/templates/e2e-dependency-nuget-publish.yml:38]: that is now int he 'outputs' section...

Comment (Saadnajmi) [reply .ado/templates/e2e-dependency-nuget-publish.yml:38]: I see... interesting.

**Review by Saadnajmi — APPROVED** [review]

**Pull request merged**